In [26]:
!pip install pymed
!pip install mechanize
!pip install html2text
!pip install pandas

     |████████████████████████████████| 45 kB 863 kB/s eta 0:00:01
     |████████████████████████████████| 115 kB 2.0 MB/s eta 0:00:01


In [4]:
import pandas as pd
import requests
import io
import pymed
import mechanize
import re
import time
import sys
import html2text
import http.cookiejar
import numpy as np

In [27]:
from google.colab import drive
drive.mount('/content/gdrive')

ModuleNotFoundError: No module named 'google'

# 1. Download all genes dataset from HGNC with OMIM ID

In [5]:
url = 'https://www.genenames.org/cgi-bin/download/custom?col=gd_hgnc_id&col=gd_app_sym&col=gd_app_name&col=gd_status&col=gd_prev_sym&col=gd_aliases&col=gd_pub_chrom_map&col=gd_pub_acc_ids&col=gd_pub_refseq_ids&col=md_mim_id&status=Approved&status=Entry%20Withdrawn&hgnc_dbtag=on&order_by=gd_app_sym_sort&format=text&submit=submit'

r = requests.get(url)
if r.ok:
    data = r.content.decode('utf8')
    df = pd.read_csv(io.StringIO(data), sep='\t')

df.shape

(47461, 10)

In [6]:
df[df['OMIM ID(supplied by OMIM)'].notnull()].shape

(16304, 10)

We hane 47461 genes and 16304 of them have OMIM ID

In [7]:
df.to_csv('gdrive/MyDrive/Биоинформатика/ИБ/Проект/Data/1_genenames.csv', index=False)
# df.to_csv('Data/1_genenames.csv', index=False)

# 2. Parse PubMed ID for genes which are in HGMD and OMIM

We create parcer which enter name of gene on http://www.hgmd.org/

If gene available it search OMIM link to gene and find all of the PubMed IDs for these gene

## 2.1. Login

In [9]:
hgmd_login_url = "http://www.hgmd.cf.ac.uk/docs/login.html"
email_address = "nokonstantinovskiy@edu.hse.ru"
password = "HGMD144550"

def initialize_browser():
    browser = mechanize.Browser()
    # Cookie Jar
    cj = http.cookiejar.LWPCookieJar()
    browser.set_cookiejar(cj)

    # Browser options
    browser.set_handle_equiv(True)
#    br.set_handle_gzip(True)
    browser.set_handle_redirect(True)
    browser.set_handle_referer(True)
    browser.set_handle_robots(False)

    # Follows refresh 0 but not hangs on refresh > 0
    browser.set_handle_refresh(mechanize._http.HTTPRefreshProcessor(), max_time=1)

    # Want debugging messages?
#    br.set_debug_http(True)
#    br.set_debug_redirects(True)
#    br.set_debug_responses(True)

    # User-Agent (this is cheating, ok?)
    browser.addheaders = [('User-agent', 'Mozilla/5.0 (X11; U; Linux i686; en-US; rv:1.9.0.1) Gecko/2008071615 Fedora/3.0.1-1.fc9 Firefox/3.0.1')]
    browser.addheaders.append(('email', email_address))

    return browser


def login_hgmd(browser):
    H2T = html2text.HTML2Text()

    response = browser.open(hgmd_login_url)
    html = response.read()

    # print response to STDOUT for debugging purposes
    # the html2text library is used for formatting the output in a more readable form
    print('Response:')
    print(html)
    print('---------')

    # print all the forms in the current page
    print('Forms:')
    print([f for f in browser.forms()])
    print('---------')

    # select login form
    browser.select_form(nr=0)
    print('Login form:')
    print(browser.form)
    print('---------')

    # print all controls in the current form, for debugging purposes
    print('Form controls:')
    print([c.name for c in browser.form.controls])
    print('---------')

    # set username and password
    browser.form['email'] = email_address
    browser.form['password'] = password

    # submit form
    response_form = browser.submit()

    # Now, you should have successfully logged in. The contents of the page will be changed. Check the contents of br.read()
    html_response = response_form.read()
    print('Login form response:')
    print(html_response)
    print('---------')

    # wait 2 seconds to not overload the server
    time.sleep(2)

    return browser

br = initialize_browser()
br = login_hgmd(br)

Response:
b'<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">\n<html>\n<head>\n<link rel="shortcut icon" href="bar/hgmd_new.ico">\n<meta http-equiv="content-type" content="text/html; charset=UTF-8">\n<title>HGMD&reg; login</title>\n<link rel="stylesheet" type="text/css" href="../docs/login.css">\n</head>\n<body bgcolor="#FFFFFF">\n\n<table border=0 width="95%" cellpadding="1" align=center><tr> \n    <td align=left valign=middle width="15%"><a href="http://www.hgmd.cf.ac.uk/"><img src="../ac/bar/hgmd_logo.gif" alt="HGMD, Cardiff" border="0"></a></td>\n    <td align=center valign=middle><h3 align=center>HGMD&reg; Login</h3></td>\n    <td align=right valign=middle width="15%"><a href="http://www.biobase-international.com"><img src="../ac/bar/biobase_source.gif" alt="Biobase" border="0"></a></td>\n</table>\n<hr width="95%">\n\n<table border="0" cellpadding="1" width="95%" align=center>\n<tr><td>\n\n<table class="login" border=0 width="100%" cellpadding="3" align=center>\n<tr>

## 2.2. Parce

We will iterate over all genes with OMIM ID and will find all PubMed articles

In [10]:
df_omim = df[df['OMIM ID(supplied by OMIM)'].notnull()]
df_omim.head()

HGNC ID Approved symbol  \
0       HGNC:5            A1BG   
2   HGNC:24086            A1CF   
4       HGNC:7             A2M   
6   HGNC:23336           A2ML1   
13  HGNC:18149          A4GALT   

                                      Approved name    Status  \
0                            alpha-1-B glycoprotein  Approved   
2                    APOBEC1 complementation factor  Approved   
4                             alpha-2-macroglobulin  Approved   
6                      alpha-2-macroglobulin like 1  Approved   
13  alpha 1,4-galactosyltransferase (P blood group)  Approved   

   Previous symbols                      Alias symbols Chromosome  \
0               NaN                                NaN   19q13.43   
2               NaN  ACF, ASP, ACF64, ACF65, APOBEC1CF   10q11.23   
4               NaN             FWP007, S863-7, CPAMD5   12p13.31   
6            CPAMD9                     FLJ25179, p170   12p13.31   
13               P1                A14GALT, Gb3S, P(k)    22q13.2   

           Accession numbers RefSeq IDs OMIM ID(supplied by OMIM)  
0                        NaN  NM_130786                    138670  
2                   AF271790  NM_014576                    618199  
4   BX647329, X68728, M11313  NM_000014                    103950  
6                   AK057908  NM_144670                    610627  
13                       NaN  NM_017436                    607922

In [22]:
search_ngmd_url = 'http://www.hgmd.cf.ac.uk/ac/index.php'
gene_url = 'http://www.hgmd.cf.ac.uk/ac/gene.php?gene={}'

cur = 0
total = df_omim.shape[0]

genes_in_hgmd = 0
pubmed_articles_num = 0

# omim link pattern
pattern = re.compile('(http:\/\/omim\.org\/entry\/[0-9]*)')

# PubMed id pattern
pubmed_p = re.compile('\[PubMed\: ([0-9]*)\]')

# flag = False

result_pubmed_dict = {'gene_symb': [], 'omim_id': [], 'pubmed_id': []}

for (i, row) in df_omim.iterrows():
    print('Processed:', cur, '/', total, 'Genes in HGMD:', genes_in_hgmd, 'PubMed Articles Num:', pubmed_articles_num, )

    symb = row['Approved symbol']

    response = br.open(gene_url.format(symb))
    html = response.read().decode('utf-8')

    omim_links = pattern.findall(html)

    if len(omim_links) > 0:
        genes_in_hgmd += 1
#         flag = True
        omim_link = omim_links[0]
        
        response = br.open(omim_link)
        html = response.read().decode('utf-8')

        pubmed_ids = pubmed_p.findall(html)

        pubmed_articles_num += len(pubmed_ids)
        
        for pid in pubmed_ids:
            result_pubmed_dict['gene_symb'].append(symb)
            result_pubmed_dict['omim_id'].append(int(omim_link.split('/')[-1]))
            result_pubmed_dict['pubmed_id'].append(int(pid))
    
#     if flag:
#         break

    time.sleep(1)

    cur += 1

Processed: 0 / 16304 Genes in HGMD: 0 PubMed Articles Num: 0
Processed: 1 / 16304 Genes in HGMD: 0 PubMed Articles Num: 0
Processed: 2 / 16304 Genes in HGMD: 0 PubMed Articles Num: 0
Processed: 3 / 16304 Genes in HGMD: 1 PubMed Articles Num: 28
Processed: 4 / 16304 Genes in HGMD: 2 PubMed Articles Num: 32
Processed: 5 / 16304 Genes in HGMD: 3 PubMed Articles Num: 49
Processed: 6 / 16304 Genes in HGMD: 3 PubMed Articles Num: 49
Processed: 7 / 16304 Genes in HGMD: 4 PubMed Articles Num: 62
Processed: 8 / 16304 Genes in HGMD: 4 PubMed Articles Num: 62
Processed: 9 / 16304 Genes in HGMD: 5 PubMed Articles Num: 68
Processed: 10 / 16304 Genes in HGMD: 5 PubMed Articles Num: 68
Processed: 11 / 16304 Genes in HGMD: 6 PubMed Articles Num: 72
Processed: 12 / 16304 Genes in HGMD: 6 PubMed Articles Num: 72
Processed: 13 / 16304 Genes in HGMD: 6 PubMed Articles Num: 72
Processed: 14 / 16304 Genes in HGMD: 7 PubMed Articles Num: 86
Processed: 15 / 16304 Genes in HGMD: 7 PubMed Articles Num: 86
Proce

Processed: 127 / 16304 Genes in HGMD: 71 PubMed Articles Num: 1030
Processed: 128 / 16304 Genes in HGMD: 72 PubMed Articles Num: 1033
Processed: 129 / 16304 Genes in HGMD: 73 PubMed Articles Num: 1053
Processed: 130 / 16304 Genes in HGMD: 73 PubMed Articles Num: 1053
Processed: 131 / 16304 Genes in HGMD: 74 PubMed Articles Num: 1093
Processed: 132 / 16304 Genes in HGMD: 75 PubMed Articles Num: 1100
Processed: 133 / 16304 Genes in HGMD: 75 PubMed Articles Num: 1100
Processed: 134 / 16304 Genes in HGMD: 75 PubMed Articles Num: 1100
Processed: 135 / 16304 Genes in HGMD: 76 PubMed Articles Num: 1110
Processed: 136 / 16304 Genes in HGMD: 77 PubMed Articles Num: 1112
Processed: 137 / 16304 Genes in HGMD: 78 PubMed Articles Num: 1130
Processed: 138 / 16304 Genes in HGMD: 79 PubMed Articles Num: 1147
Processed: 139 / 16304 Genes in HGMD: 79 PubMed Articles Num: 1147
Processed: 140 / 16304 Genes in HGMD: 79 PubMed Articles Num: 1147
Processed: 141 / 16304 Genes in HGMD: 79 PubMed Articles Num: 

Processed: 249 / 16304 Genes in HGMD: 128 PubMed Articles Num: 1704
Processed: 250 / 16304 Genes in HGMD: 129 PubMed Articles Num: 1706
Processed: 251 / 16304 Genes in HGMD: 130 PubMed Articles Num: 1710
Processed: 252 / 16304 Genes in HGMD: 131 PubMed Articles Num: 1715
Processed: 253 / 16304 Genes in HGMD: 132 PubMed Articles Num: 1716
Processed: 254 / 16304 Genes in HGMD: 132 PubMed Articles Num: 1716
Processed: 255 / 16304 Genes in HGMD: 132 PubMed Articles Num: 1716
Processed: 256 / 16304 Genes in HGMD: 133 PubMed Articles Num: 1723
Processed: 257 / 16304 Genes in HGMD: 134 PubMed Articles Num: 1725
Processed: 258 / 16304 Genes in HGMD: 135 PubMed Articles Num: 1735
Processed: 259 / 16304 Genes in HGMD: 135 PubMed Articles Num: 1735
Processed: 260 / 16304 Genes in HGMD: 135 PubMed Articles Num: 1735
Processed: 261 / 16304 Genes in HGMD: 136 PubMed Articles Num: 1758
Processed: 262 / 16304 Genes in HGMD: 137 PubMed Articles Num: 1769
Processed: 263 / 16304 Genes in HGMD: 137 PubMed

Processed: 370 / 16304 Genes in HGMD: 192 PubMed Articles Num: 2387
Processed: 371 / 16304 Genes in HGMD: 192 PubMed Articles Num: 2387
Processed: 372 / 16304 Genes in HGMD: 193 PubMed Articles Num: 2389
Processed: 373 / 16304 Genes in HGMD: 194 PubMed Articles Num: 2392
Processed: 374 / 16304 Genes in HGMD: 195 PubMed Articles Num: 2400
Processed: 375 / 16304 Genes in HGMD: 195 PubMed Articles Num: 2400
Processed: 376 / 16304 Genes in HGMD: 195 PubMed Articles Num: 2400
Processed: 377 / 16304 Genes in HGMD: 196 PubMed Articles Num: 2403
Processed: 378 / 16304 Genes in HGMD: 197 PubMed Articles Num: 2411
Processed: 379 / 16304 Genes in HGMD: 198 PubMed Articles Num: 2430
Processed: 380 / 16304 Genes in HGMD: 198 PubMed Articles Num: 2430
Processed: 381 / 16304 Genes in HGMD: 199 PubMed Articles Num: 2431
Processed: 382 / 16304 Genes in HGMD: 200 PubMed Articles Num: 2440
Processed: 383 / 16304 Genes in HGMD: 201 PubMed Articles Num: 2474
Processed: 384 / 16304 Genes in HGMD: 201 PubMed

Processed: 491 / 16304 Genes in HGMD: 261 PubMed Articles Num: 3405
Processed: 492 / 16304 Genes in HGMD: 261 PubMed Articles Num: 3405
Processed: 493 / 16304 Genes in HGMD: 261 PubMed Articles Num: 3405
Processed: 494 / 16304 Genes in HGMD: 262 PubMed Articles Num: 3409
Processed: 495 / 16304 Genes in HGMD: 263 PubMed Articles Num: 3417
Processed: 496 / 16304 Genes in HGMD: 264 PubMed Articles Num: 3425
Processed: 497 / 16304 Genes in HGMD: 265 PubMed Articles Num: 3433
Processed: 498 / 16304 Genes in HGMD: 265 PubMed Articles Num: 3433
Processed: 499 / 16304 Genes in HGMD: 265 PubMed Articles Num: 3433
Processed: 500 / 16304 Genes in HGMD: 266 PubMed Articles Num: 3434
Processed: 501 / 16304 Genes in HGMD: 267 PubMed Articles Num: 3449
Processed: 502 / 16304 Genes in HGMD: 268 PubMed Articles Num: 3461
Processed: 503 / 16304 Genes in HGMD: 269 PubMed Articles Num: 3493
Processed: 504 / 16304 Genes in HGMD: 269 PubMed Articles Num: 3493
Processed: 505 / 16304 Genes in HGMD: 270 PubMed

Processed: 612 / 16304 Genes in HGMD: 329 PubMed Articles Num: 4142
Processed: 613 / 16304 Genes in HGMD: 329 PubMed Articles Num: 4142
Processed: 614 / 16304 Genes in HGMD: 330 PubMed Articles Num: 4148
Processed: 615 / 16304 Genes in HGMD: 330 PubMed Articles Num: 4148
Processed: 616 / 16304 Genes in HGMD: 331 PubMed Articles Num: 4151
Processed: 617 / 16304 Genes in HGMD: 331 PubMed Articles Num: 4151
Processed: 618 / 16304 Genes in HGMD: 331 PubMed Articles Num: 4151
Processed: 619 / 16304 Genes in HGMD: 332 PubMed Articles Num: 4157
Processed: 620 / 16304 Genes in HGMD: 332 PubMed Articles Num: 4157
Processed: 621 / 16304 Genes in HGMD: 333 PubMed Articles Num: 4158
Processed: 622 / 16304 Genes in HGMD: 333 PubMed Articles Num: 4158
Processed: 623 / 16304 Genes in HGMD: 333 PubMed Articles Num: 4158
Processed: 624 / 16304 Genes in HGMD: 333 PubMed Articles Num: 4158
Processed: 625 / 16304 Genes in HGMD: 333 PubMed Articles Num: 4158
Processed: 626 / 16304 Genes in HGMD: 333 PubMed

Processed: 733 / 16304 Genes in HGMD: 377 PubMed Articles Num: 4643
Processed: 734 / 16304 Genes in HGMD: 378 PubMed Articles Num: 4651
Processed: 735 / 16304 Genes in HGMD: 379 PubMed Articles Num: 4657
Processed: 736 / 16304 Genes in HGMD: 379 PubMed Articles Num: 4657
Processed: 737 / 16304 Genes in HGMD: 379 PubMed Articles Num: 4657
Processed: 738 / 16304 Genes in HGMD: 379 PubMed Articles Num: 4657
Processed: 739 / 16304 Genes in HGMD: 380 PubMed Articles Num: 4748
Processed: 740 / 16304 Genes in HGMD: 381 PubMed Articles Num: 4766
Processed: 741 / 16304 Genes in HGMD: 382 PubMed Articles Num: 4784
Processed: 742 / 16304 Genes in HGMD: 383 PubMed Articles Num: 4797
Processed: 743 / 16304 Genes in HGMD: 384 PubMed Articles Num: 4927
Processed: 744 / 16304 Genes in HGMD: 384 PubMed Articles Num: 4927
Processed: 745 / 16304 Genes in HGMD: 384 PubMed Articles Num: 4927
Processed: 746 / 16304 Genes in HGMD: 384 PubMed Articles Num: 4927
Processed: 747 / 16304 Genes in HGMD: 385 PubMed

Processed: 854 / 16304 Genes in HGMD: 432 PubMed Articles Num: 6124
Processed: 855 / 16304 Genes in HGMD: 432 PubMed Articles Num: 6124
Processed: 856 / 16304 Genes in HGMD: 432 PubMed Articles Num: 6124
Processed: 857 / 16304 Genes in HGMD: 433 PubMed Articles Num: 6128
Processed: 858 / 16304 Genes in HGMD: 433 PubMed Articles Num: 6128
Processed: 859 / 16304 Genes in HGMD: 433 PubMed Articles Num: 6128
Processed: 860 / 16304 Genes in HGMD: 434 PubMed Articles Num: 6132
Processed: 861 / 16304 Genes in HGMD: 434 PubMed Articles Num: 6132
Processed: 862 / 16304 Genes in HGMD: 435 PubMed Articles Num: 6140
Processed: 863 / 16304 Genes in HGMD: 436 PubMed Articles Num: 6146
Processed: 864 / 16304 Genes in HGMD: 437 PubMed Articles Num: 6154
Processed: 865 / 16304 Genes in HGMD: 438 PubMed Articles Num: 6159
Processed: 866 / 16304 Genes in HGMD: 439 PubMed Articles Num: 6160
Processed: 867 / 16304 Genes in HGMD: 440 PubMed Articles Num: 6164
Processed: 868 / 16304 Genes in HGMD: 441 PubMed

Processed: 975 / 16304 Genes in HGMD: 480 PubMed Articles Num: 6525
Processed: 976 / 16304 Genes in HGMD: 480 PubMed Articles Num: 6525
Processed: 977 / 16304 Genes in HGMD: 481 PubMed Articles Num: 6529
Processed: 978 / 16304 Genes in HGMD: 481 PubMed Articles Num: 6529
Processed: 979 / 16304 Genes in HGMD: 481 PubMed Articles Num: 6529
Processed: 980 / 16304 Genes in HGMD: 481 PubMed Articles Num: 6529
Processed: 981 / 16304 Genes in HGMD: 481 PubMed Articles Num: 6529
Processed: 982 / 16304 Genes in HGMD: 482 PubMed Articles Num: 6537
Processed: 983 / 16304 Genes in HGMD: 482 PubMed Articles Num: 6537
Processed: 984 / 16304 Genes in HGMD: 483 PubMed Articles Num: 6538
Processed: 985 / 16304 Genes in HGMD: 484 PubMed Articles Num: 6556
Processed: 986 / 16304 Genes in HGMD: 484 PubMed Articles Num: 6556
Processed: 987 / 16304 Genes in HGMD: 484 PubMed Articles Num: 6556
Processed: 988 / 16304 Genes in HGMD: 484 PubMed Articles Num: 6556
Processed: 989 / 16304 Genes in HGMD: 484 PubMed

Processed: 1095 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1096 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1097 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1098 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1099 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1100 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1101 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1102 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1103 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1104 / 16304 Genes in HGMD: 541 PubMed Articles Num: 7264
Processed: 1105 / 16304 Genes in HGMD: 542 PubMed Articles Num: 7276
Processed: 1106 / 16304 Genes in HGMD: 543 PubMed Articles Num: 7288
Processed: 1107 / 16304 Genes in HGMD: 544 PubMed Articles Num: 7295
Processed: 1108 / 16304 Genes in HGMD: 545 PubMed Articles Num: 7303
Processed: 1109 / 16304 Genes in H

Processed: 1214 / 16304 Genes in HGMD: 602 PubMed Articles Num: 8073
Processed: 1215 / 16304 Genes in HGMD: 603 PubMed Articles Num: 8079
Processed: 1216 / 16304 Genes in HGMD: 603 PubMed Articles Num: 8079
Processed: 1217 / 16304 Genes in HGMD: 603 PubMed Articles Num: 8079
Processed: 1218 / 16304 Genes in HGMD: 603 PubMed Articles Num: 8079
Processed: 1219 / 16304 Genes in HGMD: 604 PubMed Articles Num: 8093
Processed: 1220 / 16304 Genes in HGMD: 604 PubMed Articles Num: 8093
Processed: 1221 / 16304 Genes in HGMD: 604 PubMed Articles Num: 8093
Processed: 1222 / 16304 Genes in HGMD: 604 PubMed Articles Num: 8093
Processed: 1223 / 16304 Genes in HGMD: 604 PubMed Articles Num: 8093
Processed: 1224 / 16304 Genes in HGMD: 604 PubMed Articles Num: 8093
Processed: 1225 / 16304 Genes in HGMD: 605 PubMed Articles Num: 8105
Processed: 1226 / 16304 Genes in HGMD: 606 PubMed Articles Num: 8112
Processed: 1227 / 16304 Genes in HGMD: 607 PubMed Articles Num: 8118
Processed: 1228 / 16304 Genes in H

Processed: 1333 / 16304 Genes in HGMD: 659 PubMed Articles Num: 8856
Processed: 1334 / 16304 Genes in HGMD: 660 PubMed Articles Num: 8861
Processed: 1335 / 16304 Genes in HGMD: 661 PubMed Articles Num: 8863
Processed: 1336 / 16304 Genes in HGMD: 661 PubMed Articles Num: 8863
Processed: 1337 / 16304 Genes in HGMD: 661 PubMed Articles Num: 8863
Processed: 1338 / 16304 Genes in HGMD: 661 PubMed Articles Num: 8863
Processed: 1339 / 16304 Genes in HGMD: 661 PubMed Articles Num: 8863
Processed: 1340 / 16304 Genes in HGMD: 662 PubMed Articles Num: 8898
Processed: 1341 / 16304 Genes in HGMD: 662 PubMed Articles Num: 8898
Processed: 1342 / 16304 Genes in HGMD: 662 PubMed Articles Num: 8898
Processed: 1343 / 16304 Genes in HGMD: 662 PubMed Articles Num: 8898
Processed: 1344 / 16304 Genes in HGMD: 662 PubMed Articles Num: 8898
Processed: 1345 / 16304 Genes in HGMD: 662 PubMed Articles Num: 8898
Processed: 1346 / 16304 Genes in HGMD: 662 PubMed Articles Num: 8898
Processed: 1347 / 16304 Genes in H

Processed: 1452 / 16304 Genes in HGMD: 708 PubMed Articles Num: 9427
Processed: 1453 / 16304 Genes in HGMD: 708 PubMed Articles Num: 9427
Processed: 1454 / 16304 Genes in HGMD: 708 PubMed Articles Num: 9427
Processed: 1455 / 16304 Genes in HGMD: 709 PubMed Articles Num: 9436
Processed: 1456 / 16304 Genes in HGMD: 709 PubMed Articles Num: 9436
Processed: 1457 / 16304 Genes in HGMD: 710 PubMed Articles Num: 9532
Processed: 1458 / 16304 Genes in HGMD: 711 PubMed Articles Num: 9533
Processed: 1459 / 16304 Genes in HGMD: 712 PubMed Articles Num: 9536
Processed: 1460 / 16304 Genes in HGMD: 713 PubMed Articles Num: 9545
Processed: 1461 / 16304 Genes in HGMD: 714 PubMed Articles Num: 9734
Processed: 1462 / 16304 Genes in HGMD: 715 PubMed Articles Num: 9834
Processed: 1463 / 16304 Genes in HGMD: 716 PubMed Articles Num: 9839
Processed: 1464 / 16304 Genes in HGMD: 717 PubMed Articles Num: 9840
Processed: 1465 / 16304 Genes in HGMD: 717 PubMed Articles Num: 9840
Processed: 1466 / 16304 Genes in H

Processed: 1570 / 16304 Genes in HGMD: 756 PubMed Articles Num: 10228
Processed: 1571 / 16304 Genes in HGMD: 756 PubMed Articles Num: 10228
Processed: 1572 / 16304 Genes in HGMD: 756 PubMed Articles Num: 10228
Processed: 1573 / 16304 Genes in HGMD: 756 PubMed Articles Num: 10228
Processed: 1574 / 16304 Genes in HGMD: 756 PubMed Articles Num: 10228
Processed: 1575 / 16304 Genes in HGMD: 756 PubMed Articles Num: 10228
Processed: 1576 / 16304 Genes in HGMD: 757 PubMed Articles Num: 10229
Processed: 1577 / 16304 Genes in HGMD: 757 PubMed Articles Num: 10229
Processed: 1578 / 16304 Genes in HGMD: 758 PubMed Articles Num: 10298
Processed: 1579 / 16304 Genes in HGMD: 759 PubMed Articles Num: 10304
Processed: 1580 / 16304 Genes in HGMD: 760 PubMed Articles Num: 10306
Processed: 1581 / 16304 Genes in HGMD: 760 PubMed Articles Num: 10306
Processed: 1582 / 16304 Genes in HGMD: 761 PubMed Articles Num: 10357
Processed: 1583 / 16304 Genes in HGMD: 762 PubMed Articles Num: 10421
Processed: 1584 / 16

Processed: 1688 / 16304 Genes in HGMD: 811 PubMed Articles Num: 11059
Processed: 1689 / 16304 Genes in HGMD: 811 PubMed Articles Num: 11059
Processed: 1690 / 16304 Genes in HGMD: 812 PubMed Articles Num: 11065
Processed: 1691 / 16304 Genes in HGMD: 813 PubMed Articles Num: 11069
Processed: 1692 / 16304 Genes in HGMD: 813 PubMed Articles Num: 11069
Processed: 1693 / 16304 Genes in HGMD: 813 PubMed Articles Num: 11069
Processed: 1694 / 16304 Genes in HGMD: 813 PubMed Articles Num: 11069
Processed: 1695 / 16304 Genes in HGMD: 813 PubMed Articles Num: 11069
Processed: 1696 / 16304 Genes in HGMD: 814 PubMed Articles Num: 11107
Processed: 1697 / 16304 Genes in HGMD: 814 PubMed Articles Num: 11107
Processed: 1698 / 16304 Genes in HGMD: 815 PubMed Articles Num: 11111
Processed: 1699 / 16304 Genes in HGMD: 816 PubMed Articles Num: 11121
Processed: 1700 / 16304 Genes in HGMD: 816 PubMed Articles Num: 11121
Processed: 1701 / 16304 Genes in HGMD: 816 PubMed Articles Num: 11121
Processed: 1702 / 16

Processed: 1806 / 16304 Genes in HGMD: 861 PubMed Articles Num: 11583
Processed: 1807 / 16304 Genes in HGMD: 861 PubMed Articles Num: 11583
Processed: 1808 / 16304 Genes in HGMD: 862 PubMed Articles Num: 11592
Processed: 1809 / 16304 Genes in HGMD: 863 PubMed Articles Num: 11627
Processed: 1810 / 16304 Genes in HGMD: 863 PubMed Articles Num: 11627
Processed: 1811 / 16304 Genes in HGMD: 864 PubMed Articles Num: 11638
Processed: 1812 / 16304 Genes in HGMD: 865 PubMed Articles Num: 11649
Processed: 1813 / 16304 Genes in HGMD: 866 PubMed Articles Num: 11658
Processed: 1814 / 16304 Genes in HGMD: 866 PubMed Articles Num: 11658
Processed: 1815 / 16304 Genes in HGMD: 867 PubMed Articles Num: 11666
Processed: 1816 / 16304 Genes in HGMD: 868 PubMed Articles Num: 11672
Processed: 1817 / 16304 Genes in HGMD: 869 PubMed Articles Num: 11753
Processed: 1818 / 16304 Genes in HGMD: 869 PubMed Articles Num: 11753
Processed: 1819 / 16304 Genes in HGMD: 870 PubMed Articles Num: 11762
Processed: 1820 / 16

Processed: 1924 / 16304 Genes in HGMD: 916 PubMed Articles Num: 12170
Processed: 1925 / 16304 Genes in HGMD: 917 PubMed Articles Num: 12177
Processed: 1926 / 16304 Genes in HGMD: 917 PubMed Articles Num: 12177
Processed: 1927 / 16304 Genes in HGMD: 918 PubMed Articles Num: 12207
Processed: 1928 / 16304 Genes in HGMD: 918 PubMed Articles Num: 12207
Processed: 1929 / 16304 Genes in HGMD: 919 PubMed Articles Num: 12222
Processed: 1930 / 16304 Genes in HGMD: 919 PubMed Articles Num: 12222
Processed: 1931 / 16304 Genes in HGMD: 919 PubMed Articles Num: 12222
Processed: 1932 / 16304 Genes in HGMD: 919 PubMed Articles Num: 12222
Processed: 1933 / 16304 Genes in HGMD: 919 PubMed Articles Num: 12222
Processed: 1934 / 16304 Genes in HGMD: 920 PubMed Articles Num: 12230
Processed: 1935 / 16304 Genes in HGMD: 921 PubMed Articles Num: 12245
Processed: 1936 / 16304 Genes in HGMD: 922 PubMed Articles Num: 12251
Processed: 1937 / 16304 Genes in HGMD: 922 PubMed Articles Num: 12251
Processed: 1938 / 16

Processed: 2042 / 16304 Genes in HGMD: 972 PubMed Articles Num: 12948
Processed: 2043 / 16304 Genes in HGMD: 973 PubMed Articles Num: 12981
Processed: 2044 / 16304 Genes in HGMD: 974 PubMed Articles Num: 13016
Processed: 2045 / 16304 Genes in HGMD: 975 PubMed Articles Num: 13043
Processed: 2046 / 16304 Genes in HGMD: 975 PubMed Articles Num: 13043
Processed: 2047 / 16304 Genes in HGMD: 975 PubMed Articles Num: 13043
Processed: 2048 / 16304 Genes in HGMD: 975 PubMed Articles Num: 13043
Processed: 2049 / 16304 Genes in HGMD: 976 PubMed Articles Num: 13048
Processed: 2050 / 16304 Genes in HGMD: 977 PubMed Articles Num: 13070
Processed: 2051 / 16304 Genes in HGMD: 978 PubMed Articles Num: 13074
Processed: 2052 / 16304 Genes in HGMD: 979 PubMed Articles Num: 13100
Processed: 2053 / 16304 Genes in HGMD: 979 PubMed Articles Num: 13100
Processed: 2054 / 16304 Genes in HGMD: 979 PubMed Articles Num: 13100
Processed: 2055 / 16304 Genes in HGMD: 979 PubMed Articles Num: 13100
Processed: 2056 / 16

Processed: 2159 / 16304 Genes in HGMD: 1027 PubMed Articles Num: 13574
Processed: 2160 / 16304 Genes in HGMD: 1027 PubMed Articles Num: 13574
Processed: 2161 / 16304 Genes in HGMD: 1028 PubMed Articles Num: 13596
Processed: 2162 / 16304 Genes in HGMD: 1028 PubMed Articles Num: 13596
Processed: 2163 / 16304 Genes in HGMD: 1028 PubMed Articles Num: 13596
Processed: 2164 / 16304 Genes in HGMD: 1029 PubMed Articles Num: 13599
Processed: 2165 / 16304 Genes in HGMD: 1030 PubMed Articles Num: 13600
Processed: 2166 / 16304 Genes in HGMD: 1030 PubMed Articles Num: 13600
Processed: 2167 / 16304 Genes in HGMD: 1030 PubMed Articles Num: 13600
Processed: 2168 / 16304 Genes in HGMD: 1031 PubMed Articles Num: 13601
Processed: 2169 / 16304 Genes in HGMD: 1031 PubMed Articles Num: 13601
Processed: 2170 / 16304 Genes in HGMD: 1031 PubMed Articles Num: 13601
Processed: 2171 / 16304 Genes in HGMD: 1032 PubMed Articles Num: 13618
Processed: 2172 / 16304 Genes in HGMD: 1032 PubMed Articles Num: 13618
Proces

Processed: 2275 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2276 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2277 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2278 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2279 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2280 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2281 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2282 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2283 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2284 / 16304 Genes in HGMD: 1073 PubMed Articles Num: 14238
Processed: 2285 / 16304 Genes in HGMD: 1074 PubMed Articles Num: 14242
Processed: 2286 / 16304 Genes in HGMD: 1074 PubMed Articles Num: 14242
Processed: 2287 / 16304 Genes in HGMD: 1075 PubMed Articles Num: 14244
Processed: 2288 / 16304 Genes in HGMD: 1075 PubMed Articles Num: 14244
Proces

Processed: 2391 / 16304 Genes in HGMD: 1124 PubMed Articles Num: 14991
Processed: 2392 / 16304 Genes in HGMD: 1125 PubMed Articles Num: 14995
Processed: 2393 / 16304 Genes in HGMD: 1126 PubMed Articles Num: 15000
Processed: 2394 / 16304 Genes in HGMD: 1127 PubMed Articles Num: 15012
Processed: 2395 / 16304 Genes in HGMD: 1127 PubMed Articles Num: 15012
Processed: 2396 / 16304 Genes in HGMD: 1128 PubMed Articles Num: 15024
Processed: 2397 / 16304 Genes in HGMD: 1128 PubMed Articles Num: 15024
Processed: 2398 / 16304 Genes in HGMD: 1128 PubMed Articles Num: 15024
Processed: 2399 / 16304 Genes in HGMD: 1128 PubMed Articles Num: 15024
Processed: 2400 / 16304 Genes in HGMD: 1128 PubMed Articles Num: 15024
Processed: 2401 / 16304 Genes in HGMD: 1128 PubMed Articles Num: 15024
Processed: 2402 / 16304 Genes in HGMD: 1129 PubMed Articles Num: 15032
Processed: 2403 / 16304 Genes in HGMD: 1130 PubMed Articles Num: 15039
Processed: 2404 / 16304 Genes in HGMD: 1131 PubMed Articles Num: 15044
Proces

Processed: 2507 / 16304 Genes in HGMD: 1186 PubMed Articles Num: 15643
Processed: 2508 / 16304 Genes in HGMD: 1186 PubMed Articles Num: 15643
Processed: 2509 / 16304 Genes in HGMD: 1187 PubMed Articles Num: 15654
Processed: 2510 / 16304 Genes in HGMD: 1188 PubMed Articles Num: 15658
Processed: 2511 / 16304 Genes in HGMD: 1188 PubMed Articles Num: 15658
Processed: 2512 / 16304 Genes in HGMD: 1189 PubMed Articles Num: 15664
Processed: 2513 / 16304 Genes in HGMD: 1190 PubMed Articles Num: 15685
Processed: 2514 / 16304 Genes in HGMD: 1191 PubMed Articles Num: 15691
Processed: 2515 / 16304 Genes in HGMD: 1192 PubMed Articles Num: 15697
Processed: 2516 / 16304 Genes in HGMD: 1193 PubMed Articles Num: 15699
Processed: 2517 / 16304 Genes in HGMD: 1194 PubMed Articles Num: 15701
Processed: 2518 / 16304 Genes in HGMD: 1194 PubMed Articles Num: 15701
Processed: 2519 / 16304 Genes in HGMD: 1194 PubMed Articles Num: 15701
Processed: 2520 / 16304 Genes in HGMD: 1194 PubMed Articles Num: 15701
Proces

Processed: 2623 / 16304 Genes in HGMD: 1233 PubMed Articles Num: 16052
Processed: 2624 / 16304 Genes in HGMD: 1233 PubMed Articles Num: 16052
Processed: 2625 / 16304 Genes in HGMD: 1233 PubMed Articles Num: 16052
Processed: 2626 / 16304 Genes in HGMD: 1233 PubMed Articles Num: 16052
Processed: 2627 / 16304 Genes in HGMD: 1234 PubMed Articles Num: 16058
Processed: 2628 / 16304 Genes in HGMD: 1235 PubMed Articles Num: 16092
Processed: 2629 / 16304 Genes in HGMD: 1236 PubMed Articles Num: 16104
Processed: 2630 / 16304 Genes in HGMD: 1237 PubMed Articles Num: 16115
Processed: 2631 / 16304 Genes in HGMD: 1238 PubMed Articles Num: 16124
Processed: 2632 / 16304 Genes in HGMD: 1239 PubMed Articles Num: 16128
Processed: 2633 / 16304 Genes in HGMD: 1240 PubMed Articles Num: 16133
Processed: 2634 / 16304 Genes in HGMD: 1241 PubMed Articles Num: 16172
Processed: 2635 / 16304 Genes in HGMD: 1242 PubMed Articles Num: 16180
Processed: 2636 / 16304 Genes in HGMD: 1243 PubMed Articles Num: 16185
Proces

Processed: 2739 / 16304 Genes in HGMD: 1297 PubMed Articles Num: 16557
Processed: 2740 / 16304 Genes in HGMD: 1297 PubMed Articles Num: 16557
Processed: 2741 / 16304 Genes in HGMD: 1298 PubMed Articles Num: 16561
Processed: 2742 / 16304 Genes in HGMD: 1298 PubMed Articles Num: 16561
Processed: 2743 / 16304 Genes in HGMD: 1299 PubMed Articles Num: 16564
Processed: 2744 / 16304 Genes in HGMD: 1300 PubMed Articles Num: 16570
Processed: 2745 / 16304 Genes in HGMD: 1300 PubMed Articles Num: 16570
Processed: 2746 / 16304 Genes in HGMD: 1300 PubMed Articles Num: 16570
Processed: 2747 / 16304 Genes in HGMD: 1301 PubMed Articles Num: 16577
Processed: 2748 / 16304 Genes in HGMD: 1301 PubMed Articles Num: 16577
Processed: 2749 / 16304 Genes in HGMD: 1302 PubMed Articles Num: 16579
Processed: 2750 / 16304 Genes in HGMD: 1303 PubMed Articles Num: 16598
Processed: 2751 / 16304 Genes in HGMD: 1304 PubMed Articles Num: 16603
Processed: 2752 / 16304 Genes in HGMD: 1305 PubMed Articles Num: 16606
Proces

Processed: 2855 / 16304 Genes in HGMD: 1365 PubMed Articles Num: 17865
Processed: 2856 / 16304 Genes in HGMD: 1365 PubMed Articles Num: 17865
Processed: 2857 / 16304 Genes in HGMD: 1365 PubMed Articles Num: 17865
Processed: 2858 / 16304 Genes in HGMD: 1365 PubMed Articles Num: 17865
Processed: 2859 / 16304 Genes in HGMD: 1365 PubMed Articles Num: 17865
Processed: 2860 / 16304 Genes in HGMD: 1365 PubMed Articles Num: 17865
Processed: 2861 / 16304 Genes in HGMD: 1365 PubMed Articles Num: 17865
Processed: 2862 / 16304 Genes in HGMD: 1366 PubMed Articles Num: 17873
Processed: 2863 / 16304 Genes in HGMD: 1367 PubMed Articles Num: 17876
Processed: 2864 / 16304 Genes in HGMD: 1367 PubMed Articles Num: 17876
Processed: 2865 / 16304 Genes in HGMD: 1367 PubMed Articles Num: 17876
Processed: 2866 / 16304 Genes in HGMD: 1368 PubMed Articles Num: 17883
Processed: 2867 / 16304 Genes in HGMD: 1368 PubMed Articles Num: 17883
Processed: 2868 / 16304 Genes in HGMD: 1369 PubMed Articles Num: 17888
Proces

Processed: 2971 / 16304 Genes in HGMD: 1413 PubMed Articles Num: 18368
Processed: 2972 / 16304 Genes in HGMD: 1414 PubMed Articles Num: 18392
Processed: 2973 / 16304 Genes in HGMD: 1414 PubMed Articles Num: 18392
Processed: 2974 / 16304 Genes in HGMD: 1415 PubMed Articles Num: 18415
Processed: 2975 / 16304 Genes in HGMD: 1415 PubMed Articles Num: 18415
Processed: 2976 / 16304 Genes in HGMD: 1416 PubMed Articles Num: 18422
Processed: 2977 / 16304 Genes in HGMD: 1416 PubMed Articles Num: 18422
Processed: 2978 / 16304 Genes in HGMD: 1416 PubMed Articles Num: 18422
Processed: 2979 / 16304 Genes in HGMD: 1417 PubMed Articles Num: 18425
Processed: 2980 / 16304 Genes in HGMD: 1417 PubMed Articles Num: 18425
Processed: 2981 / 16304 Genes in HGMD: 1418 PubMed Articles Num: 18428
Processed: 2982 / 16304 Genes in HGMD: 1418 PubMed Articles Num: 18428
Processed: 2983 / 16304 Genes in HGMD: 1419 PubMed Articles Num: 18429
Processed: 2984 / 16304 Genes in HGMD: 1420 PubMed Articles Num: 18435
Proces

Processed: 3087 / 16304 Genes in HGMD: 1468 PubMed Articles Num: 19015
Processed: 3088 / 16304 Genes in HGMD: 1468 PubMed Articles Num: 19015
Processed: 3089 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3090 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3091 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3092 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3093 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3094 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3095 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3096 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3097 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3098 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3099 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Processed: 3100 / 16304 Genes in HGMD: 1469 PubMed Articles Num: 19016
Proces

Processed: 3203 / 16304 Genes in HGMD: 1521 PubMed Articles Num: 19743
Processed: 3204 / 16304 Genes in HGMD: 1522 PubMed Articles Num: 19792
Processed: 3205 / 16304 Genes in HGMD: 1522 PubMed Articles Num: 19792
Processed: 3206 / 16304 Genes in HGMD: 1522 PubMed Articles Num: 19792
Processed: 3207 / 16304 Genes in HGMD: 1522 PubMed Articles Num: 19792
Processed: 3208 / 16304 Genes in HGMD: 1522 PubMed Articles Num: 19792
Processed: 3209 / 16304 Genes in HGMD: 1522 PubMed Articles Num: 19792
Processed: 3210 / 16304 Genes in HGMD: 1523 PubMed Articles Num: 19805
Processed: 3211 / 16304 Genes in HGMD: 1523 PubMed Articles Num: 19805
Processed: 3212 / 16304 Genes in HGMD: 1523 PubMed Articles Num: 19805
Processed: 3213 / 16304 Genes in HGMD: 1523 PubMed Articles Num: 19805
Processed: 3214 / 16304 Genes in HGMD: 1524 PubMed Articles Num: 19844
Processed: 3215 / 16304 Genes in HGMD: 1525 PubMed Articles Num: 19846
Processed: 3216 / 16304 Genes in HGMD: 1525 PubMed Articles Num: 19846
Proces

Processed: 3319 / 16304 Genes in HGMD: 1593 PubMed Articles Num: 20856
Processed: 3320 / 16304 Genes in HGMD: 1593 PubMed Articles Num: 20856
Processed: 3321 / 16304 Genes in HGMD: 1593 PubMed Articles Num: 20856
Processed: 3322 / 16304 Genes in HGMD: 1594 PubMed Articles Num: 20863
Processed: 3323 / 16304 Genes in HGMD: 1595 PubMed Articles Num: 20867
Processed: 3324 / 16304 Genes in HGMD: 1595 PubMed Articles Num: 20867
Processed: 3325 / 16304 Genes in HGMD: 1595 PubMed Articles Num: 20867
Processed: 3326 / 16304 Genes in HGMD: 1595 PubMed Articles Num: 20867
Processed: 3327 / 16304 Genes in HGMD: 1596 PubMed Articles Num: 20878
Processed: 3328 / 16304 Genes in HGMD: 1596 PubMed Articles Num: 20878
Processed: 3329 / 16304 Genes in HGMD: 1597 PubMed Articles Num: 20880
Processed: 3330 / 16304 Genes in HGMD: 1598 PubMed Articles Num: 20882
Processed: 3331 / 16304 Genes in HGMD: 1598 PubMed Articles Num: 20882
Processed: 3332 / 16304 Genes in HGMD: 1599 PubMed Articles Num: 20888
Proces

Processed: 3435 / 16304 Genes in HGMD: 1651 PubMed Articles Num: 21341
Processed: 3436 / 16304 Genes in HGMD: 1652 PubMed Articles Num: 21344
Processed: 3437 / 16304 Genes in HGMD: 1652 PubMed Articles Num: 21344
Processed: 3438 / 16304 Genes in HGMD: 1652 PubMed Articles Num: 21344
Processed: 3439 / 16304 Genes in HGMD: 1652 PubMed Articles Num: 21344
Processed: 3440 / 16304 Genes in HGMD: 1653 PubMed Articles Num: 21345
Processed: 3441 / 16304 Genes in HGMD: 1653 PubMed Articles Num: 21345
Processed: 3442 / 16304 Genes in HGMD: 1653 PubMed Articles Num: 21345
Processed: 3443 / 16304 Genes in HGMD: 1653 PubMed Articles Num: 21345
Processed: 3444 / 16304 Genes in HGMD: 1653 PubMed Articles Num: 21345
Processed: 3445 / 16304 Genes in HGMD: 1654 PubMed Articles Num: 21369
Processed: 3446 / 16304 Genes in HGMD: 1655 PubMed Articles Num: 21370
Processed: 3447 / 16304 Genes in HGMD: 1655 PubMed Articles Num: 21370
Processed: 3448 / 16304 Genes in HGMD: 1655 PubMed Articles Num: 21370
Proces

Processed: 3551 / 16304 Genes in HGMD: 1689 PubMed Articles Num: 21672
Processed: 3552 / 16304 Genes in HGMD: 1689 PubMed Articles Num: 21672
Processed: 3553 / 16304 Genes in HGMD: 1690 PubMed Articles Num: 21674
Processed: 3554 / 16304 Genes in HGMD: 1691 PubMed Articles Num: 21679
Processed: 3555 / 16304 Genes in HGMD: 1692 PubMed Articles Num: 21686
Processed: 3556 / 16304 Genes in HGMD: 1693 PubMed Articles Num: 21689
Processed: 3557 / 16304 Genes in HGMD: 1693 PubMed Articles Num: 21689
Processed: 3558 / 16304 Genes in HGMD: 1693 PubMed Articles Num: 21689
Processed: 3559 / 16304 Genes in HGMD: 1694 PubMed Articles Num: 21699
Processed: 3560 / 16304 Genes in HGMD: 1695 PubMed Articles Num: 21710
Processed: 3561 / 16304 Genes in HGMD: 1696 PubMed Articles Num: 21720
Processed: 3562 / 16304 Genes in HGMD: 1696 PubMed Articles Num: 21720
Processed: 3563 / 16304 Genes in HGMD: 1697 PubMed Articles Num: 21726
Processed: 3564 / 16304 Genes in HGMD: 1698 PubMed Articles Num: 21790
Proces

Processed: 3667 / 16304 Genes in HGMD: 1761 PubMed Articles Num: 22575
Processed: 3668 / 16304 Genes in HGMD: 1762 PubMed Articles Num: 22595
Processed: 3669 / 16304 Genes in HGMD: 1763 PubMed Articles Num: 22600
Processed: 3670 / 16304 Genes in HGMD: 1764 PubMed Articles Num: 22602
Processed: 3671 / 16304 Genes in HGMD: 1764 PubMed Articles Num: 22602
Processed: 3672 / 16304 Genes in HGMD: 1765 PubMed Articles Num: 22605
Processed: 3673 / 16304 Genes in HGMD: 1766 PubMed Articles Num: 22609
Processed: 3674 / 16304 Genes in HGMD: 1766 PubMed Articles Num: 22609
Processed: 3675 / 16304 Genes in HGMD: 1766 PubMed Articles Num: 22609
Processed: 3676 / 16304 Genes in HGMD: 1767 PubMed Articles Num: 22612
Processed: 3677 / 16304 Genes in HGMD: 1767 PubMed Articles Num: 22612
Processed: 3678 / 16304 Genes in HGMD: 1767 PubMed Articles Num: 22612
Processed: 3679 / 16304 Genes in HGMD: 1768 PubMed Articles Num: 22617
Processed: 3680 / 16304 Genes in HGMD: 1768 PubMed Articles Num: 22617
Proces

Processed: 3783 / 16304 Genes in HGMD: 1815 PubMed Articles Num: 23086
Processed: 3784 / 16304 Genes in HGMD: 1816 PubMed Articles Num: 23093
Processed: 3785 / 16304 Genes in HGMD: 1816 PubMed Articles Num: 23093
Processed: 3786 / 16304 Genes in HGMD: 1816 PubMed Articles Num: 23093
Processed: 3787 / 16304 Genes in HGMD: 1816 PubMed Articles Num: 23093
Processed: 3788 / 16304 Genes in HGMD: 1817 PubMed Articles Num: 23096
Processed: 3789 / 16304 Genes in HGMD: 1817 PubMed Articles Num: 23096
Processed: 3790 / 16304 Genes in HGMD: 1817 PubMed Articles Num: 23096
Processed: 3791 / 16304 Genes in HGMD: 1817 PubMed Articles Num: 23096
Processed: 3792 / 16304 Genes in HGMD: 1817 PubMed Articles Num: 23096
Processed: 3793 / 16304 Genes in HGMD: 1817 PubMed Articles Num: 23096
Processed: 3794 / 16304 Genes in HGMD: 1817 PubMed Articles Num: 23096
Processed: 3795 / 16304 Genes in HGMD: 1817 PubMed Articles Num: 23096
Processed: 3796 / 16304 Genes in HGMD: 1818 PubMed Articles Num: 23100
Proces

Processed: 3899 / 16304 Genes in HGMD: 1860 PubMed Articles Num: 23577
Processed: 3900 / 16304 Genes in HGMD: 1860 PubMed Articles Num: 23577
Processed: 3901 / 16304 Genes in HGMD: 1861 PubMed Articles Num: 23585
Processed: 3902 / 16304 Genes in HGMD: 1862 PubMed Articles Num: 23611
Processed: 3903 / 16304 Genes in HGMD: 1863 PubMed Articles Num: 23612
Processed: 3904 / 16304 Genes in HGMD: 1863 PubMed Articles Num: 23612
Processed: 3905 / 16304 Genes in HGMD: 1863 PubMed Articles Num: 23612
Processed: 3906 / 16304 Genes in HGMD: 1863 PubMed Articles Num: 23612
Processed: 3907 / 16304 Genes in HGMD: 1864 PubMed Articles Num: 23627
Processed: 3908 / 16304 Genes in HGMD: 1864 PubMed Articles Num: 23627
Processed: 3909 / 16304 Genes in HGMD: 1864 PubMed Articles Num: 23627
Processed: 3910 / 16304 Genes in HGMD: 1865 PubMed Articles Num: 23634
Processed: 3911 / 16304 Genes in HGMD: 1865 PubMed Articles Num: 23634
Processed: 3912 / 16304 Genes in HGMD: 1865 PubMed Articles Num: 23634
Proces

Processed: 4015 / 16304 Genes in HGMD: 1909 PubMed Articles Num: 24150
Processed: 4016 / 16304 Genes in HGMD: 1909 PubMed Articles Num: 24150
Processed: 4017 / 16304 Genes in HGMD: 1910 PubMed Articles Num: 24157
Processed: 4018 / 16304 Genes in HGMD: 1910 PubMed Articles Num: 24157
Processed: 4019 / 16304 Genes in HGMD: 1911 PubMed Articles Num: 24170
Processed: 4020 / 16304 Genes in HGMD: 1912 PubMed Articles Num: 24193
Processed: 4021 / 16304 Genes in HGMD: 1912 PubMed Articles Num: 24193
Processed: 4022 / 16304 Genes in HGMD: 1912 PubMed Articles Num: 24193
Processed: 4023 / 16304 Genes in HGMD: 1913 PubMed Articles Num: 24196
Processed: 4024 / 16304 Genes in HGMD: 1913 PubMed Articles Num: 24196
Processed: 4025 / 16304 Genes in HGMD: 1913 PubMed Articles Num: 24196
Processed: 4026 / 16304 Genes in HGMD: 1914 PubMed Articles Num: 24203
Processed: 4027 / 16304 Genes in HGMD: 1915 PubMed Articles Num: 24220
Processed: 4028 / 16304 Genes in HGMD: 1915 PubMed Articles Num: 24220
Proces

Processed: 4131 / 16304 Genes in HGMD: 1952 PubMed Articles Num: 24635
Processed: 4132 / 16304 Genes in HGMD: 1953 PubMed Articles Num: 24677
Processed: 4133 / 16304 Genes in HGMD: 1954 PubMed Articles Num: 24682
Processed: 4134 / 16304 Genes in HGMD: 1954 PubMed Articles Num: 24682
Processed: 4135 / 16304 Genes in HGMD: 1954 PubMed Articles Num: 24682
Processed: 4136 / 16304 Genes in HGMD: 1954 PubMed Articles Num: 24682
Processed: 4137 / 16304 Genes in HGMD: 1954 PubMed Articles Num: 24682
Processed: 4138 / 16304 Genes in HGMD: 1954 PubMed Articles Num: 24682
Processed: 4139 / 16304 Genes in HGMD: 1954 PubMed Articles Num: 24682
Processed: 4140 / 16304 Genes in HGMD: 1955 PubMed Articles Num: 24697
Processed: 4141 / 16304 Genes in HGMD: 1956 PubMed Articles Num: 24699
Processed: 4142 / 16304 Genes in HGMD: 1956 PubMed Articles Num: 24699
Processed: 4143 / 16304 Genes in HGMD: 1956 PubMed Articles Num: 24699
Processed: 4144 / 16304 Genes in HGMD: 1957 PubMed Articles Num: 24719
Proces

Processed: 4247 / 16304 Genes in HGMD: 2002 PubMed Articles Num: 25205
Processed: 4248 / 16304 Genes in HGMD: 2003 PubMed Articles Num: 25208
Processed: 4249 / 16304 Genes in HGMD: 2004 PubMed Articles Num: 25226
Processed: 4250 / 16304 Genes in HGMD: 2005 PubMed Articles Num: 25241
Processed: 4251 / 16304 Genes in HGMD: 2005 PubMed Articles Num: 25241
Processed: 4252 / 16304 Genes in HGMD: 2005 PubMed Articles Num: 25241
Processed: 4253 / 16304 Genes in HGMD: 2006 PubMed Articles Num: 25263
Processed: 4254 / 16304 Genes in HGMD: 2006 PubMed Articles Num: 25263
Processed: 4255 / 16304 Genes in HGMD: 2006 PubMed Articles Num: 25263
Processed: 4256 / 16304 Genes in HGMD: 2006 PubMed Articles Num: 25263
Processed: 4257 / 16304 Genes in HGMD: 2006 PubMed Articles Num: 25263
Processed: 4258 / 16304 Genes in HGMD: 2007 PubMed Articles Num: 25307
Processed: 4259 / 16304 Genes in HGMD: 2007 PubMed Articles Num: 25307
Processed: 4260 / 16304 Genes in HGMD: 2008 PubMed Articles Num: 25336
Proces

Processed: 4363 / 16304 Genes in HGMD: 2054 PubMed Articles Num: 26012
Processed: 4364 / 16304 Genes in HGMD: 2054 PubMed Articles Num: 26012
Processed: 4365 / 16304 Genes in HGMD: 2054 PubMed Articles Num: 26012
Processed: 4366 / 16304 Genes in HGMD: 2055 PubMed Articles Num: 26020
Processed: 4367 / 16304 Genes in HGMD: 2056 PubMed Articles Num: 26027
Processed: 4368 / 16304 Genes in HGMD: 2057 PubMed Articles Num: 26063
Processed: 4369 / 16304 Genes in HGMD: 2057 PubMed Articles Num: 26063
Processed: 4370 / 16304 Genes in HGMD: 2057 PubMed Articles Num: 26063
Processed: 4371 / 16304 Genes in HGMD: 2058 PubMed Articles Num: 26079
Processed: 4372 / 16304 Genes in HGMD: 2059 PubMed Articles Num: 26098
Processed: 4373 / 16304 Genes in HGMD: 2059 PubMed Articles Num: 26098
Processed: 4374 / 16304 Genes in HGMD: 2059 PubMed Articles Num: 26098
Processed: 4375 / 16304 Genes in HGMD: 2060 PubMed Articles Num: 26100
Processed: 4376 / 16304 Genes in HGMD: 2060 PubMed Articles Num: 26100
Proces

Processed: 4479 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4480 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4481 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4482 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4483 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4484 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4485 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4486 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4487 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4488 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4489 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4490 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4491 / 16304 Genes in HGMD: 2114 PubMed Articles Num: 27140
Processed: 4492 / 16304 Genes in HGMD: 2115 PubMed Articles Num: 27147
Proces

Processed: 4595 / 16304 Genes in HGMD: 2161 PubMed Articles Num: 27752
Processed: 4596 / 16304 Genes in HGMD: 2161 PubMed Articles Num: 27752
Processed: 4597 / 16304 Genes in HGMD: 2161 PubMed Articles Num: 27752
Processed: 4598 / 16304 Genes in HGMD: 2161 PubMed Articles Num: 27752
Processed: 4599 / 16304 Genes in HGMD: 2161 PubMed Articles Num: 27752
Processed: 4600 / 16304 Genes in HGMD: 2161 PubMed Articles Num: 27752
Processed: 4601 / 16304 Genes in HGMD: 2162 PubMed Articles Num: 27753
Processed: 4602 / 16304 Genes in HGMD: 2163 PubMed Articles Num: 27754
Processed: 4603 / 16304 Genes in HGMD: 2164 PubMed Articles Num: 27755
Processed: 4604 / 16304 Genes in HGMD: 2164 PubMed Articles Num: 27755
Processed: 4605 / 16304 Genes in HGMD: 2164 PubMed Articles Num: 27755
Processed: 4606 / 16304 Genes in HGMD: 2164 PubMed Articles Num: 27755
Processed: 4607 / 16304 Genes in HGMD: 2165 PubMed Articles Num: 27762
Processed: 4608 / 16304 Genes in HGMD: 2165 PubMed Articles Num: 27762
Proces

Processed: 4711 / 16304 Genes in HGMD: 2209 PubMed Articles Num: 28187
Processed: 4712 / 16304 Genes in HGMD: 2210 PubMed Articles Num: 28208
Processed: 4713 / 16304 Genes in HGMD: 2211 PubMed Articles Num: 28228
Processed: 4714 / 16304 Genes in HGMD: 2211 PubMed Articles Num: 28228
Processed: 4715 / 16304 Genes in HGMD: 2211 PubMed Articles Num: 28228
Processed: 4716 / 16304 Genes in HGMD: 2212 PubMed Articles Num: 28237
Processed: 4717 / 16304 Genes in HGMD: 2212 PubMed Articles Num: 28237
Processed: 4718 / 16304 Genes in HGMD: 2213 PubMed Articles Num: 28238
Processed: 4719 / 16304 Genes in HGMD: 2214 PubMed Articles Num: 28254
Processed: 4720 / 16304 Genes in HGMD: 2215 PubMed Articles Num: 28282
Processed: 4721 / 16304 Genes in HGMD: 2216 PubMed Articles Num: 28296
Processed: 4722 / 16304 Genes in HGMD: 2216 PubMed Articles Num: 28296
Processed: 4723 / 16304 Genes in HGMD: 2217 PubMed Articles Num: 28304
Processed: 4724 / 16304 Genes in HGMD: 2217 PubMed Articles Num: 28304
Proces

Processed: 4827 / 16304 Genes in HGMD: 2276 PubMed Articles Num: 29527
Processed: 4828 / 16304 Genes in HGMD: 2276 PubMed Articles Num: 29527
Processed: 4829 / 16304 Genes in HGMD: 2277 PubMed Articles Num: 29568
Processed: 4830 / 16304 Genes in HGMD: 2278 PubMed Articles Num: 29571
Processed: 4831 / 16304 Genes in HGMD: 2279 PubMed Articles Num: 29575
Processed: 4832 / 16304 Genes in HGMD: 2280 PubMed Articles Num: 29578
Processed: 4833 / 16304 Genes in HGMD: 2280 PubMed Articles Num: 29578
Processed: 4834 / 16304 Genes in HGMD: 2281 PubMed Articles Num: 29582
Processed: 4835 / 16304 Genes in HGMD: 2281 PubMed Articles Num: 29582
Processed: 4836 / 16304 Genes in HGMD: 2281 PubMed Articles Num: 29582
Processed: 4837 / 16304 Genes in HGMD: 2281 PubMed Articles Num: 29582
Processed: 4838 / 16304 Genes in HGMD: 2281 PubMed Articles Num: 29582
Processed: 4839 / 16304 Genes in HGMD: 2281 PubMed Articles Num: 29582
Processed: 4840 / 16304 Genes in HGMD: 2282 PubMed Articles Num: 29587
Proces

Processed: 4943 / 16304 Genes in HGMD: 2334 PubMed Articles Num: 30242
Processed: 4944 / 16304 Genes in HGMD: 2334 PubMed Articles Num: 30242
Processed: 4945 / 16304 Genes in HGMD: 2334 PubMed Articles Num: 30242
Processed: 4946 / 16304 Genes in HGMD: 2334 PubMed Articles Num: 30242
Processed: 4947 / 16304 Genes in HGMD: 2335 PubMed Articles Num: 30247
Processed: 4948 / 16304 Genes in HGMD: 2335 PubMed Articles Num: 30247
Processed: 4949 / 16304 Genes in HGMD: 2336 PubMed Articles Num: 30274
Processed: 4950 / 16304 Genes in HGMD: 2337 PubMed Articles Num: 30275
Processed: 4951 / 16304 Genes in HGMD: 2338 PubMed Articles Num: 30314
Processed: 4952 / 16304 Genes in HGMD: 2339 PubMed Articles Num: 30319
Processed: 4953 / 16304 Genes in HGMD: 2340 PubMed Articles Num: 30334
Processed: 4954 / 16304 Genes in HGMD: 2341 PubMed Articles Num: 30340
Processed: 4955 / 16304 Genes in HGMD: 2341 PubMed Articles Num: 30340
Processed: 4956 / 16304 Genes in HGMD: 2341 PubMed Articles Num: 30340
Proces

Processed: 5059 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5060 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5061 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5062 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5063 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5064 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5065 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5066 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5067 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5068 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5069 / 16304 Genes in HGMD: 2386 PubMed Articles Num: 31253
Processed: 5070 / 16304 Genes in HGMD: 2387 PubMed Articles Num: 31254
Processed: 5071 / 16304 Genes in HGMD: 2388 PubMed Articles Num: 31278
Processed: 5072 / 16304 Genes in HGMD: 2388 PubMed Articles Num: 31278
Proces

Processed: 5175 / 16304 Genes in HGMD: 2446 PubMed Articles Num: 32114
Processed: 5176 / 16304 Genes in HGMD: 2446 PubMed Articles Num: 32114
Processed: 5177 / 16304 Genes in HGMD: 2446 PubMed Articles Num: 32114
Processed: 5178 / 16304 Genes in HGMD: 2446 PubMed Articles Num: 32114
Processed: 5179 / 16304 Genes in HGMD: 2447 PubMed Articles Num: 32123
Processed: 5180 / 16304 Genes in HGMD: 2447 PubMed Articles Num: 32123
Processed: 5181 / 16304 Genes in HGMD: 2447 PubMed Articles Num: 32123
Processed: 5182 / 16304 Genes in HGMD: 2447 PubMed Articles Num: 32123
Processed: 5183 / 16304 Genes in HGMD: 2447 PubMed Articles Num: 32123
Processed: 5184 / 16304 Genes in HGMD: 2448 PubMed Articles Num: 32131
Processed: 5185 / 16304 Genes in HGMD: 2448 PubMed Articles Num: 32131
Processed: 5186 / 16304 Genes in HGMD: 2449 PubMed Articles Num: 32152
Processed: 5187 / 16304 Genes in HGMD: 2449 PubMed Articles Num: 32152
Processed: 5188 / 16304 Genes in HGMD: 2449 PubMed Articles Num: 32152
Proces

Processed: 5291 / 16304 Genes in HGMD: 2502 PubMed Articles Num: 33087
Processed: 5292 / 16304 Genes in HGMD: 2503 PubMed Articles Num: 33105
Processed: 5293 / 16304 Genes in HGMD: 2504 PubMed Articles Num: 33112
Processed: 5294 / 16304 Genes in HGMD: 2504 PubMed Articles Num: 33112
Processed: 5295 / 16304 Genes in HGMD: 2505 PubMed Articles Num: 33133
Processed: 5296 / 16304 Genes in HGMD: 2505 PubMed Articles Num: 33133
Processed: 5297 / 16304 Genes in HGMD: 2505 PubMed Articles Num: 33133
Processed: 5298 / 16304 Genes in HGMD: 2506 PubMed Articles Num: 33149
Processed: 5299 / 16304 Genes in HGMD: 2507 PubMed Articles Num: 33156
Processed: 5300 / 16304 Genes in HGMD: 2508 PubMed Articles Num: 33161
Processed: 5301 / 16304 Genes in HGMD: 2508 PubMed Articles Num: 33161
Processed: 5302 / 16304 Genes in HGMD: 2509 PubMed Articles Num: 33161
Processed: 5303 / 16304 Genes in HGMD: 2510 PubMed Articles Num: 33177
Processed: 5304 / 16304 Genes in HGMD: 2510 PubMed Articles Num: 33177
Proces

Processed: 5407 / 16304 Genes in HGMD: 2559 PubMed Articles Num: 33918
Processed: 5408 / 16304 Genes in HGMD: 2559 PubMed Articles Num: 33918
Processed: 5409 / 16304 Genes in HGMD: 2559 PubMed Articles Num: 33918
Processed: 5410 / 16304 Genes in HGMD: 2559 PubMed Articles Num: 33918
Processed: 5411 / 16304 Genes in HGMD: 2559 PubMed Articles Num: 33918
Processed: 5412 / 16304 Genes in HGMD: 2559 PubMed Articles Num: 33918
Processed: 5413 / 16304 Genes in HGMD: 2560 PubMed Articles Num: 33926
Processed: 5414 / 16304 Genes in HGMD: 2561 PubMed Articles Num: 33937
Processed: 5415 / 16304 Genes in HGMD: 2561 PubMed Articles Num: 33937
Processed: 5416 / 16304 Genes in HGMD: 2561 PubMed Articles Num: 33937
Processed: 5417 / 16304 Genes in HGMD: 2561 PubMed Articles Num: 33937
Processed: 5418 / 16304 Genes in HGMD: 2562 PubMed Articles Num: 33956
Processed: 5419 / 16304 Genes in HGMD: 2563 PubMed Articles Num: 33961
Processed: 5420 / 16304 Genes in HGMD: 2564 PubMed Articles Num: 33987
Proces

Processed: 5523 / 16304 Genes in HGMD: 2601 PubMed Articles Num: 34303
Processed: 5524 / 16304 Genes in HGMD: 2601 PubMed Articles Num: 34303
Processed: 5525 / 16304 Genes in HGMD: 2602 PubMed Articles Num: 34308
Processed: 5526 / 16304 Genes in HGMD: 2603 PubMed Articles Num: 34309
Processed: 5527 / 16304 Genes in HGMD: 2603 PubMed Articles Num: 34309
Processed: 5528 / 16304 Genes in HGMD: 2603 PubMed Articles Num: 34309
Processed: 5529 / 16304 Genes in HGMD: 2603 PubMed Articles Num: 34309
Processed: 5530 / 16304 Genes in HGMD: 2603 PubMed Articles Num: 34309
Processed: 5531 / 16304 Genes in HGMD: 2604 PubMed Articles Num: 34313
Processed: 5532 / 16304 Genes in HGMD: 2604 PubMed Articles Num: 34313
Processed: 5533 / 16304 Genes in HGMD: 2604 PubMed Articles Num: 34313
Processed: 5534 / 16304 Genes in HGMD: 2604 PubMed Articles Num: 34313
Processed: 5535 / 16304 Genes in HGMD: 2604 PubMed Articles Num: 34313
Processed: 5536 / 16304 Genes in HGMD: 2605 PubMed Articles Num: 34321
Proces

Processed: 5639 / 16304 Genes in HGMD: 2657 PubMed Articles Num: 34905
Processed: 5640 / 16304 Genes in HGMD: 2657 PubMed Articles Num: 34905
Processed: 5641 / 16304 Genes in HGMD: 2658 PubMed Articles Num: 34917
Processed: 5642 / 16304 Genes in HGMD: 2658 PubMed Articles Num: 34917
Processed: 5643 / 16304 Genes in HGMD: 2658 PubMed Articles Num: 34917
Processed: 5644 / 16304 Genes in HGMD: 2659 PubMed Articles Num: 34920
Processed: 5645 / 16304 Genes in HGMD: 2660 PubMed Articles Num: 34921
Processed: 5646 / 16304 Genes in HGMD: 2660 PubMed Articles Num: 34921
Processed: 5647 / 16304 Genes in HGMD: 2660 PubMed Articles Num: 34921
Processed: 5648 / 16304 Genes in HGMD: 2661 PubMed Articles Num: 34930
Processed: 5649 / 16304 Genes in HGMD: 2661 PubMed Articles Num: 34930
Processed: 5650 / 16304 Genes in HGMD: 2662 PubMed Articles Num: 34935
Processed: 5651 / 16304 Genes in HGMD: 2663 PubMed Articles Num: 34938
Processed: 5652 / 16304 Genes in HGMD: 2663 PubMed Articles Num: 34938
Proces

Processed: 5755 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5756 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5757 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5758 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5759 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5760 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5761 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5762 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5763 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5764 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5765 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5766 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5767 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Processed: 5768 / 16304 Genes in HGMD: 2713 PubMed Articles Num: 35475
Proces

Processed: 5871 / 16304 Genes in HGMD: 2731 PubMed Articles Num: 35707
Processed: 5872 / 16304 Genes in HGMD: 2732 PubMed Articles Num: 35711
Processed: 5873 / 16304 Genes in HGMD: 2733 PubMed Articles Num: 35715
Processed: 5874 / 16304 Genes in HGMD: 2734 PubMed Articles Num: 35720
Processed: 5875 / 16304 Genes in HGMD: 2735 PubMed Articles Num: 35734
Processed: 5876 / 16304 Genes in HGMD: 2736 PubMed Articles Num: 35746
Processed: 5877 / 16304 Genes in HGMD: 2737 PubMed Articles Num: 35757
Processed: 5878 / 16304 Genes in HGMD: 2738 PubMed Articles Num: 36142
Processed: 5879 / 16304 Genes in HGMD: 2739 PubMed Articles Num: 36296
Processed: 5880 / 16304 Genes in HGMD: 2740 PubMed Articles Num: 37380
Processed: 5881 / 16304 Genes in HGMD: 2741 PubMed Articles Num: 37450
Processed: 5882 / 16304 Genes in HGMD: 2742 PubMed Articles Num: 37461
Processed: 5883 / 16304 Genes in HGMD: 2743 PubMed Articles Num: 37477
Processed: 5884 / 16304 Genes in HGMD: 2744 PubMed Articles Num: 37564
Proces

Processed: 5987 / 16304 Genes in HGMD: 2796 PubMed Articles Num: 38265
Processed: 5988 / 16304 Genes in HGMD: 2796 PubMed Articles Num: 38265
Processed: 5989 / 16304 Genes in HGMD: 2796 PubMed Articles Num: 38265
Processed: 5990 / 16304 Genes in HGMD: 2796 PubMed Articles Num: 38265
Processed: 5991 / 16304 Genes in HGMD: 2796 PubMed Articles Num: 38265
Processed: 5992 / 16304 Genes in HGMD: 2797 PubMed Articles Num: 38272
Processed: 5993 / 16304 Genes in HGMD: 2797 PubMed Articles Num: 38272
Processed: 5994 / 16304 Genes in HGMD: 2797 PubMed Articles Num: 38272
Processed: 5995 / 16304 Genes in HGMD: 2798 PubMed Articles Num: 38273
Processed: 5996 / 16304 Genes in HGMD: 2799 PubMed Articles Num: 38339
Processed: 5997 / 16304 Genes in HGMD: 2799 PubMed Articles Num: 38339
Processed: 5998 / 16304 Genes in HGMD: 2799 PubMed Articles Num: 38339
Processed: 5999 / 16304 Genes in HGMD: 2800 PubMed Articles Num: 38341
Processed: 6000 / 16304 Genes in HGMD: 2800 PubMed Articles Num: 38341
Proces

Processed: 6103 / 16304 Genes in HGMD: 2855 PubMed Articles Num: 39274
Processed: 6104 / 16304 Genes in HGMD: 2855 PubMed Articles Num: 39274
Processed: 6105 / 16304 Genes in HGMD: 2856 PubMed Articles Num: 39277
Processed: 6106 / 16304 Genes in HGMD: 2856 PubMed Articles Num: 39277
Processed: 6107 / 16304 Genes in HGMD: 2857 PubMed Articles Num: 39286
Processed: 6108 / 16304 Genes in HGMD: 2857 PubMed Articles Num: 39286
Processed: 6109 / 16304 Genes in HGMD: 2858 PubMed Articles Num: 39288
Processed: 6110 / 16304 Genes in HGMD: 2858 PubMed Articles Num: 39288
Processed: 6111 / 16304 Genes in HGMD: 2858 PubMed Articles Num: 39288
Processed: 6112 / 16304 Genes in HGMD: 2858 PubMed Articles Num: 39288
Processed: 6113 / 16304 Genes in HGMD: 2858 PubMed Articles Num: 39288
Processed: 6114 / 16304 Genes in HGMD: 2859 PubMed Articles Num: 39291
Processed: 6115 / 16304 Genes in HGMD: 2859 PubMed Articles Num: 39291
Processed: 6116 / 16304 Genes in HGMD: 2860 PubMed Articles Num: 39294
Proces

Processed: 6219 / 16304 Genes in HGMD: 2909 PubMed Articles Num: 40038
Processed: 6220 / 16304 Genes in HGMD: 2909 PubMed Articles Num: 40038
Processed: 6221 / 16304 Genes in HGMD: 2910 PubMed Articles Num: 40062
Processed: 6222 / 16304 Genes in HGMD: 2910 PubMed Articles Num: 40062
Processed: 6223 / 16304 Genes in HGMD: 2910 PubMed Articles Num: 40062
Processed: 6224 / 16304 Genes in HGMD: 2911 PubMed Articles Num: 40063
Processed: 6225 / 16304 Genes in HGMD: 2911 PubMed Articles Num: 40063
Processed: 6226 / 16304 Genes in HGMD: 2911 PubMed Articles Num: 40063
Processed: 6227 / 16304 Genes in HGMD: 2911 PubMed Articles Num: 40063
Processed: 6228 / 16304 Genes in HGMD: 2912 PubMed Articles Num: 40067
Processed: 6229 / 16304 Genes in HGMD: 2912 PubMed Articles Num: 40067
Processed: 6230 / 16304 Genes in HGMD: 2913 PubMed Articles Num: 40072
Processed: 6231 / 16304 Genes in HGMD: 2914 PubMed Articles Num: 40073
Processed: 6232 / 16304 Genes in HGMD: 2914 PubMed Articles Num: 40073
Proces

Processed: 6335 / 16304 Genes in HGMD: 2971 PubMed Articles Num: 40773
Processed: 6336 / 16304 Genes in HGMD: 2971 PubMed Articles Num: 40773
Processed: 6337 / 16304 Genes in HGMD: 2971 PubMed Articles Num: 40773
Processed: 6338 / 16304 Genes in HGMD: 2971 PubMed Articles Num: 40773
Processed: 6339 / 16304 Genes in HGMD: 2971 PubMed Articles Num: 40773
Processed: 6340 / 16304 Genes in HGMD: 2972 PubMed Articles Num: 40793
Processed: 6341 / 16304 Genes in HGMD: 2972 PubMed Articles Num: 40793
Processed: 6342 / 16304 Genes in HGMD: 2973 PubMed Articles Num: 40823
Processed: 6343 / 16304 Genes in HGMD: 2974 PubMed Articles Num: 40863
Processed: 6344 / 16304 Genes in HGMD: 2974 PubMed Articles Num: 40863
Processed: 6345 / 16304 Genes in HGMD: 2975 PubMed Articles Num: 40868
Processed: 6346 / 16304 Genes in HGMD: 2975 PubMed Articles Num: 40868
Processed: 6347 / 16304 Genes in HGMD: 2975 PubMed Articles Num: 40868
Processed: 6348 / 16304 Genes in HGMD: 2975 PubMed Articles Num: 40868
Proces

Processed: 6451 / 16304 Genes in HGMD: 3019 PubMed Articles Num: 41522
Processed: 6452 / 16304 Genes in HGMD: 3020 PubMed Articles Num: 41530
Processed: 6453 / 16304 Genes in HGMD: 3020 PubMed Articles Num: 41530
Processed: 6454 / 16304 Genes in HGMD: 3021 PubMed Articles Num: 41539
Processed: 6455 / 16304 Genes in HGMD: 3022 PubMed Articles Num: 41541
Processed: 6456 / 16304 Genes in HGMD: 3022 PubMed Articles Num: 41541
Processed: 6457 / 16304 Genes in HGMD: 3022 PubMed Articles Num: 41541
Processed: 6458 / 16304 Genes in HGMD: 3023 PubMed Articles Num: 41543
Processed: 6459 / 16304 Genes in HGMD: 3023 PubMed Articles Num: 41543
Processed: 6460 / 16304 Genes in HGMD: 3023 PubMed Articles Num: 41543
Processed: 6461 / 16304 Genes in HGMD: 3024 PubMed Articles Num: 41544
Processed: 6462 / 16304 Genes in HGMD: 3024 PubMed Articles Num: 41544
Processed: 6463 / 16304 Genes in HGMD: 3025 PubMed Articles Num: 41582
Processed: 6464 / 16304 Genes in HGMD: 3025 PubMed Articles Num: 41582
Proces

Processed: 6567 / 16304 Genes in HGMD: 3092 PubMed Articles Num: 42793
Processed: 6568 / 16304 Genes in HGMD: 3093 PubMed Articles Num: 42795
Processed: 6569 / 16304 Genes in HGMD: 3094 PubMed Articles Num: 42801
Processed: 6570 / 16304 Genes in HGMD: 3094 PubMed Articles Num: 42801


KeyboardInterrupt: 

In [32]:
result_pubmed_df = pd.DataFrame(result_pubmed_dict)

result_pubmed_df.to_csv('gdrive/MyDrive/Биоинформатика/ИБ/Проект/Data/2_true_pubmedids_part1.csv', index=False)
# result_pubmed_df.to_csv('Data/2_true_pubmedids_part1.csv', index=False)

In [33]:
print('Number of genes:', len(result_pubmed_df.gene_symb.unique()))
print('Number of PubMed Articles:', len(result_pubmed_df.pubmed_id.unique()))

Number of genes: 3089
Number of PubMed Articles: 38451


# 3. Download titles and abstract from PubMed

## 3.1. Download random PubMed articles

In [46]:
# PubMed min and max id to download random samples
pubmed_min_id = 1
pubmed_max_id = 33297669

negative_class_size = 200000

# PubMed settings
tool_name = 'MyTool'
pubmed_query = 'https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pubmed&id={}&tool={}&email={}'

# title pattern
title_pattern = re.compile('title[\s]*{[\s]*name[\s]*\"([^\}]+)')

# abstract pattern
abstract_pattern = re.compile('abstract[\s]*\"([^\"]+)')

def get_pubmed_title_and_abstract(id):
    title = None
    abstract = None

    r = requests.get(pubmed_query.format(id, tool_name, email_address))
    clean_text = r.text.replace('\n', '').replace('\t', '')
    titles = title_pattern.findall(clean_text)

    if len(titles) > 0:
        title = titles[0][:titles[0].rfind('"')]

        abstracts = abstract_pattern.findall(clean_text)
        if len(abstracts) > 0 and abstracts[0] is not None:
            abstract = abstracts[0]
    
    time.sleep(0.2)

    # print(id, title, abstract)

    return title, abstract

In [42]:
negative_ids = list(np.random.choice(pubmed_max_id - pubmed_min_id, negative_class_size, replace=False) + pubmed_min_id + 1)

results = { 'pubmed_id': [], 'title': [], 'abstract': [] }

cur = 0
real_num = 0

for i in range(len(negative_ids)):
    print('Processed:', cur, '/', negative_class_size, 'Not Nones:', real_num)

    id = negative_ids[i]
    res = get_pubmed_title_and_abstract(id)
    if res[0] is not None:
        results['pubmed_id'].append(id)
        results['title'].append(res[0])
        results['abstract'].append(res[1])
        real_num += 1

    cur += 1

NameError: name 'np' is not defined

In [ ]:
false_articles_df = pd.DataFrame(results)

false_articles_df.loc[false_articles_df.abstract.isna(), 'abstract'] = ''
false_articles_df.title = false_articles_df.title.str.split('",', n=1, expand=True)[0]
false_articles_df.title = false_articles_df.title.str.lstrip('[').str.rstrip('].')

false_articles_df.to_csv('gdrive/MyDrive/Биоинформатика/ИБ/Проект/Data/3_false_articles.csv', index=False)
# false_articles_df.to_csv('Data/3_false_articles.csv', index=False)

## 3.2. Download true PubMed articles

In [47]:
positive_ids = list(result_pubmed_df.pubmed_id.unique())

true_results = { 'pubmed_id': [], 'title': [], 'abstract': [] }

cur = 0
real_num = 0

for i in range(len(positive_ids)):
    print('Processed:', cur, '/', len(positive_ids), 'Not Nones:', real_num)

    id = positive_ids[i]
    res = get_pubmed_title_and_abstract(id)
    if res[0] is not None:
        true_results['pubmed_id'].append(id)
        true_results['title'].append(res[0])
        true_results['abstract'].append(res[1])
        real_num += 1

    cur += 1

Processed: 0 / 38451 Not Nones: 0
Processed: 1 / 38451 Not Nones: 1
Processed: 2 / 38451 Not Nones: 2
Processed: 3 / 38451 Not Nones: 3
Processed: 4 / 38451 Not Nones: 4
Processed: 5 / 38451 Not Nones: 5
Processed: 6 / 38451 Not Nones: 6
Processed: 7 / 38451 Not Nones: 7
Processed: 8 / 38451 Not Nones: 8
Processed: 9 / 38451 Not Nones: 9
Processed: 10 / 38451 Not Nones: 10
Processed: 11 / 38451 Not Nones: 11
Processed: 12 / 38451 Not Nones: 12
Processed: 13 / 38451 Not Nones: 13
Processed: 14 / 38451 Not Nones: 14
Processed: 15 / 38451 Not Nones: 15
Processed: 16 / 38451 Not Nones: 16
Processed: 17 / 38451 Not Nones: 17
Processed: 18 / 38451 Not Nones: 18
Processed: 19 / 38451 Not Nones: 19
Processed: 20 / 38451 Not Nones: 20
Processed: 21 / 38451 Not Nones: 21
Processed: 22 / 38451 Not Nones: 22
Processed: 23 / 38451 Not Nones: 23
Processed: 24 / 38451 Not Nones: 24
Processed: 25 / 38451 Not Nones: 25
Processed: 26 / 38451 Not Nones: 26
Processed: 27 / 38451 Not Nones: 27
Processed: 2

Processed: 222 / 38451 Not Nones: 222
Processed: 223 / 38451 Not Nones: 223
Processed: 224 / 38451 Not Nones: 224
Processed: 225 / 38451 Not Nones: 225
Processed: 226 / 38451 Not Nones: 226
Processed: 227 / 38451 Not Nones: 227
Processed: 228 / 38451 Not Nones: 228
Processed: 229 / 38451 Not Nones: 229
Processed: 230 / 38451 Not Nones: 230
Processed: 231 / 38451 Not Nones: 231
Processed: 232 / 38451 Not Nones: 232
Processed: 233 / 38451 Not Nones: 233
Processed: 234 / 38451 Not Nones: 234
Processed: 235 / 38451 Not Nones: 235
Processed: 236 / 38451 Not Nones: 236
Processed: 237 / 38451 Not Nones: 237
Processed: 238 / 38451 Not Nones: 238
Processed: 239 / 38451 Not Nones: 239
Processed: 240 / 38451 Not Nones: 240
Processed: 241 / 38451 Not Nones: 241
Processed: 242 / 38451 Not Nones: 242
Processed: 243 / 38451 Not Nones: 243
Processed: 244 / 38451 Not Nones: 244
Processed: 245 / 38451 Not Nones: 245
Processed: 246 / 38451 Not Nones: 246
Processed: 247 / 38451 Not Nones: 247
Processed: 2

Processed: 438 / 38451 Not Nones: 438
Processed: 439 / 38451 Not Nones: 439
Processed: 440 / 38451 Not Nones: 440
Processed: 441 / 38451 Not Nones: 441
Processed: 442 / 38451 Not Nones: 442
Processed: 443 / 38451 Not Nones: 443
Processed: 444 / 38451 Not Nones: 444
Processed: 445 / 38451 Not Nones: 445
Processed: 446 / 38451 Not Nones: 446
Processed: 447 / 38451 Not Nones: 447
Processed: 448 / 38451 Not Nones: 448
Processed: 449 / 38451 Not Nones: 449
Processed: 450 / 38451 Not Nones: 450
Processed: 451 / 38451 Not Nones: 451
Processed: 452 / 38451 Not Nones: 452
Processed: 453 / 38451 Not Nones: 453
Processed: 454 / 38451 Not Nones: 454
Processed: 455 / 38451 Not Nones: 455
Processed: 456 / 38451 Not Nones: 456
Processed: 457 / 38451 Not Nones: 457
Processed: 458 / 38451 Not Nones: 458
Processed: 459 / 38451 Not Nones: 459
Processed: 460 / 38451 Not Nones: 460
Processed: 461 / 38451 Not Nones: 461
Processed: 462 / 38451 Not Nones: 462
Processed: 463 / 38451 Not Nones: 463
Processed: 4

Processed: 654 / 38451 Not Nones: 654
Processed: 655 / 38451 Not Nones: 655
Processed: 656 / 38451 Not Nones: 656
Processed: 657 / 38451 Not Nones: 657
Processed: 658 / 38451 Not Nones: 658
Processed: 659 / 38451 Not Nones: 659
Processed: 660 / 38451 Not Nones: 660
Processed: 661 / 38451 Not Nones: 661
Processed: 662 / 38451 Not Nones: 662
Processed: 663 / 38451 Not Nones: 663
Processed: 664 / 38451 Not Nones: 664
Processed: 665 / 38451 Not Nones: 665
Processed: 666 / 38451 Not Nones: 666
Processed: 667 / 38451 Not Nones: 667
Processed: 668 / 38451 Not Nones: 668
Processed: 669 / 38451 Not Nones: 669
Processed: 670 / 38451 Not Nones: 670
Processed: 671 / 38451 Not Nones: 671
Processed: 672 / 38451 Not Nones: 672
Processed: 673 / 38451 Not Nones: 673
Processed: 674 / 38451 Not Nones: 674
Processed: 675 / 38451 Not Nones: 675
Processed: 676 / 38451 Not Nones: 676
Processed: 677 / 38451 Not Nones: 677
Processed: 678 / 38451 Not Nones: 678
Processed: 679 / 38451 Not Nones: 679
Processed: 6

KeyboardInterrupt: 

In [54]:
true_articles_df = pd.DataFrame(true_results)

true_articles_df.loc[true_articles_df.abstract.isna(), 'abstract'] = ''
true_articles_df.title = true_articles_df.title.str.split('",', n=1, expand=True)[0]
true_articles_df.title = true_articles_df.title.str.lstrip('[').str.rstrip('].')

true_articles_df.to_csv('gdrive/MyDrive/Биоинформатика/ИБ/Проект/Data/3_true_articles.csv', index=False)
# true_articles_df.to_csv('Data/3_true_articles.csv', index=False)

## 3.3. Remove articles from false df if it in true df

In [73]:
true_articles_df = pd.read_csv('gdrive/MyDrive/Биоинформатика/ИБ/Проект/Data/3_true_articles.csv').fillna('')
false_articles_df = pd.read_csv('gdrive/MyDrive/Биоинформатика/ИБ/Проект/Data/3_false_articles.csv').fillna('')
# true_articles_df = pd.read_csv('Data/3_true_articles.csv').fillna('')
# false_articles_df = pd.read_csv('Data/3_false_articles.csv').fillna('')

false_articles_df = false_articles_df[~false_articles_df.pubmed_id.isin(positive_ids)]

In [74]:
print('true_articles_df:', true_articles_df.shape[0])
print('false_articles_df:', false_articles_df.shape[0])

true_articles_df: 13923
false_articles_df: 19690


## 3.4. Concat DataSets

In [101]:
true_articles_df['label'] = int(True)
false_articles_df['label'] = int(False)

all_articles_df = pd.concat([true_articles_df, false_articles_df]).reset_index(drop=True)

In [102]:
# all_articles_df.to_csv('gdrive/MyDrive/Биоинформатика/ИБ/Проект/Data/3_all_articles.csv', index=False)
all_articles_df.to_csv('Data/3_all_articles.csv', index=False)

## 3.5. Create validation set

In [103]:
all_articles_df.label.sum() / all_articles_df.shape[0]

0.41421473834528305

In [104]:
test = all_articles_df.sample(2000)
test.label.sum() / test.shape[0]

0.413

In [105]:
all_articles_df.shape

(33613, 4)

In [107]:
TRAIN_articles_df = all_articles_df.drop(test.index)

In [108]:
TRAIN_articles_df.to_csv('gdrive/MyDrive/Биоинформатика/ИБ/Проект/Data/3_TRAIN_articles.csv', index=False)
test.to_csv('gdrive/MyDrive/Биоинформатика/ИБ/Проект/Data/3_TEST_articles.csv', index=False)
# TRAIN_articles_df.to_csv('Data/3_TRAIN_articles.csv', index=False)
# test.to_csv('Data/3_TEST_articles.csv', index=False)